CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 3.1.6
+  - Added docs to help people avoid concurrency issues (often caused by accidentally relying on shared state with global variables, constants, or unguarded instance variables) [#58](https://github.com/logstash-plugins/logstash-filter-ruby/issues/58)
```

Comment:
```suggestion
  - [DOC] Added docs to help people avoid concurrency issues (often caused by accidentally relying on shared state with global variables, constants, or unguarded instance variables) [#58](https://github.com/logstash-plugins/logstash-filter-ruby/issues/58)
```

Reply:
For consistency with the changelog guidelines and other plugin items in release notes